lib/sqlalchemy/dialects/postgresql/asyncpg.py

```diff
@@ -28,7 +28,7 @@
 an ad-hoc event loop.  This mode of operation is of **limited use**
 and is for special testing scenarios only.  The mode can be enabled by
 adding the SQLAlchemy-specific flag ``async_fallback`` to the URL
-in conjunction with :func:`_sa.craete_engine`::
+in conjunction with :func:`_sa.create_engine`::
```

Comment:
I'm somewhat surprised that sphinx didn't catch this as a no-member error as there's no documented `craete_engine` `:func:`

Reply:
it probably logs is as warning, but I think there are too many to triage so they are mostly ignored :)